src/index_runner/main.py

```diff
@@ -30,8 +30,20 @@
 def _handle_msg(msg):
     event_type = msg.get('evtype')
     if not event_type:
-        logger.warning(f"Missing 'evtype' in event: {msg}")
-        return
+        msg = f"Missing 'evtype' in event: {msg}"
```

Comment:
We don't really need to `raise` here, doesn't make much difference in runtime as the event loop will just catch and continue. Mostly I did this so we can more easily test this function